omas/examples/fuse_data_export.py

```diff
@@ -0,0 +1,97 @@
+import time
+import omas
+import sys
+from omas.omas_utils import printe
+from omas.machine_mappings import d3d
+from numpy import *
+import argparse
+
+
+def fuse_export(save_path, device, shot, EFIT_TREE, PROFILES_TREE, 
+                EFIT_RUN_ID, PROFILES_RUN_ID, CER_analysis_type="CERQUICK"):
+    ods = omas.ODS()
+    
+    if len(EFIT_RUN_ID) == 0:
+        EFIT_RUN_ID = None
+    if len(PROFILES_RUN_ID) == 0:
+        PROFILES_RUN_ID = None
+    
+    tic = time.time()
+    if device.lower() != "d3d":
+        raise ValueError(f"Unsupported device {device}. Only 'd3d' supported at present.")
+    printe("- Fetching ec_launcher data")
+    d3d.ec_launcher_active_hardware(ods, shot)
+
+    # printe("- Fetching nbi data")
+    # d3d.nbi_active_hardware(ods, shot)
+
+    printe("- Fetching core_profiles data")
+    d3d.core_profiles_profile_1d(ods, shot, PROFILES_tree=PROFILES_TREE, 
+                                 PROFILES_run_id=PROFILES_RUN_ID)
+
+    printe("- Fetching wall data")
+    d3d.wall(ods, shot)
+
```

Comment:
```suggestion

        printe("- Fetching magnetic hardware data")
        d3d.magnetics_hardware(ods, $shot)

```
Include missing lines from the FUSE master branch